backend/private-graph/graph/resolver.go

```diff
@@ -474,9 +474,6 @@ func (r *Resolver) isAdminInProject(ctx context.Context, project_id int) (*model
 
 	span.SetTag("ProjectID", project_id)
 
-	if util.IsTestEnv() {
-		return nil, nil
-	}
```

Comment:
this was to bypass the auth context, but i'm changing the tests to leverage the simple auth mode instead and not need this mock.